README.md

```diff
@@ -320,6 +323,34 @@ tube_evaluator = TubeEvaluator(annot_filepath, preds_filepath)
+|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|---------|------------------------------------------------------------------------------------------------------------------------|
+| --help, -h      | Shows a help message for tool                                                                                                                         | python cli.py -h                                             |         |                                                                                                                        |
+| --anno_gt       | Takes in a directory or file to the ground-truth annotations                                                                                          | python cli.py --anno_gt toyexample/gts_vocpascal_format      |         | Based on the `gtformat`, make sure this parameter points to the right kind of ground truth (directory or file)         |
+| --anno_gt       | Takes in a directory or file to the detection annotations                                                                                             | python cli.py --anno_gt toyexample/dets_classname_rel_xyx2y2 |         | Based on the `detformat`, make sure this parameter points to the right kind of ground truth (directory or file)        |
+| --img           | Takes in a directory where the corresponding images are located                                                                                       | python cli.py --img toyexample/images                        |         | Used in some ground truth annotation formats. At the current moment, there is visualization functionality for the CLI  |
+| --gtformat      | Takes in keyword that represents the format that the groundtruth will be in (example: COCO GT JSON format)                                            | python cli.py --gtformat coco                                |         | Available formats: `coco`, `voc`, `imagenet`, `labelme`, `openimg`, `yolo`, `absolute`, `cvat`, `tube`                 |
+| --detformat     | Takes in keyword that represents the format that the detection will be in (either text files or COCO Detection JSON format)                           | python cli.py --detformat coco                               |         | Available inputs: `coco`, `xyrb` (x,y,x2,y2), or `xywh` (x,y,width,height)                                             |
+| --metrics       | Takes in keyword that represents the metric that will be used                                                                                         | python cli.py --metrics voc2012                              |         |                                                                                                                        |
```

Comment:
another possible option would be `coco`, right?
Shouldn't `voc2012` be in the **Default** column?
Shouldn't `coco` be in the **Notes** column?

Reply:
Yes you're right, there is no options for `--metrics` in the **Notes** column. I'll add all three options. Currently, the CLI requires the user to define a metric; otherwise it will error out. That's why there is no default value in the doc table. If you want there to be a default value, I'm fine with that.

Reply:
Metric options added in https://github.com/rafaelpadilla/review_object_detection_metrics/pull/57/commits/b7cb0e9e74c1f3f27e90eb83a61db86a3798605c